### ISSUE TITLE
[bug] The Styles Submenu Items Lack Aria Labels 

### ISSUE THREAD

**Issue by KDSBrowne** [tldraw/tldraw-v1#65]
The styles submenu color items lack any labels, announcing only “menu item # of 12” to screen readers, thus failing to identify the current color. 

![image](https://user-images.githubusercontent.com/22058534/207055829-7aa5006c-2802-488e-975e-d92df2faba8f.png)

Recommendation: Provide a unique label for each color that describes the color.
